Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -140,18 +140,13 @@ try {
         }
     }
     $removeFiles = @()
-#    $dstFolder = Join-Path $ENV:GITHUB_WORKSPACE ".github\workflows"
-#    $pathLength = "$ENV:GITHUB_WORKSPACE".Length
-#    'ci.yaml','cd.yaml','registercustomerenvironment.yaml','ReleaseWorkflowTemplate.yaml.txt','ReleaseTo*-*-*-*-*-*-*.yaml' | ForEach-Object {
-#        $removeFiles += @(Get-Item (Join-Path $dstFolder $_) -ErrorAction SilentlyContinue | ForEach-Object { $_.FullName.Substring($pathLength) })
-#    }
 
     if (-not $update) {
         if (($updateFiles) -or ($removeFiles)) {
-            OutputWarning -message "Updated AL-Go System Files are available for your repository, please run the Update AL-Go System Files workflow"
+            OutputWarning -message "There are updates for your AL-Go system, run 'Update AL-Go System Files' workflow to download the latest version of AL-Go."
         }
         else {
-            Write-Host "No updated AL-Go System Files are available"
+            Write-Host "Your repository runs on the latest version of AL-Go System."
         }
```

Comment:
Technically it is wrong to call it the latest version of AL-Go System
A repo that was created using v0.1 might be up-to-date with v0.1, but a v0.2 exists.
This means that he would have the latest files in v0.1 but a newer version is available (not yet detected by the script)

Reply:
I also thought about it. but if scripts is not detecting any new update it means that they run on the latest. IMO we should update the script to detect a new branch and report it?